src/main/java/nl/fuchsia/controller/ZaakController.java

```diff
@@ -42,8 +43,8 @@ public Zaak getZaakById(@PathVariable("zaakNr") Integer zaakNr) {
         return zaakService.getZaakById(zaakNr);
     }
 
-    @GetMapping(value = "zaken?persoonnr={persoonnr}")
-    public ResponseEntity<List<Zaak>> getZakenByPersoon(@PathVariable("persoonnr") Integer persoonnr) {
+    @GetMapping(value = "zaken?persoonnr")
```

Comment:
De basepath van deze controller is al /zaken, toch? Dan wordt dit /zaken/zaken?persoonnr.
Maar ?persoonnr mag ook wel weg. Dat je dat verwacht wordt al geïmpliceerd door @RequestParam (bij default is een gegeven url/request param required).

Dit gaat er voor zorgen dat je een conflict krijgt met de standaard @GetMapping.
Dus misschien dat je deze twee methodes moet fuseren tot één.

Reply:
Basepath /zaken/zaken?persoonnr kent hij niet, ik kan alleen opvragen via /zaken?persoonnr=nr.

Ik had ook geprobeerd de value weg te halen, daarna kreeg ik wel een conflict.

Reply:
/zaken?persoonnr={nr} komt uit op /zaken (de originele). Bij /zaken/zaken?persoonnr probeert die naar /zaken/{zaaknr} te gaan.
Het is mij niet gelukt om deze methode met zaken?persoonnr te raken.

Reply:
Ok, jetty werkte weer niet goed, heb nu de twee samengevoegd, maar ik moet testen nog herschrijven, is dit hoe je het bedoelde?

Reply:
Ook de test aangepast

Reply:
Testen zijn ook aangepast

Reply:
Ja, al zou ik denk ik checken voor null en de waarde op niet required zetten i.p.v. de defaultwaarde gebruiken, al zal je niet snel een persoon met ID 0 hebben, dus dit werkt ook prima.

Reply:
Ok, dat ook nog aangepast.